v2/internal/binding/generate_test.go

```diff
@@ -111,18 +111,28 @@ func Test_goTypeToJSDocType(t *testing.T) {
 		{
 			name:  "anything else",
 			input: "foo",
-			want:  "any",
+			want:  "foo",
```

Comment:
This seems like a change with major potential impact, so let me know I should revert it.